security/src/main/java/io/micronaut/security/authentication/DefaultAuthorizationExceptionHandler.java

```diff
@@ -45,23 +47,33 @@
 
     private final RedirectConfiguration redirectConfiguration;
     private final PriorToLoginPersistence priorToLoginPersistence;
+    private final ErrorResponseProcessor<?> responseProcessor;
 
     /**
      * Default constructor.
      */
     public DefaultAuthorizationExceptionHandler() {
-        this.redirectConfiguration = null;
-        this.priorToLoginPersistence = null;
+        this(null, null, null);
     }
 
     /**
      * @param redirectConfiguration Redirect configuration
```

Comment:
add javadoc `@deprecated` poiting the user to alternative.

Reply:
Done